superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -295,7 +296,30 @@ export default class FilterableTable extends PureComponent<
     sortBy: string;
     sortDirection: SortDirectionType;
   }) {
-    this.setState({ sortBy, sortDirection });
+    this.setState(
```

Comment:
this is a lot to put in a setState, could we make the sort direction into its own function and pass it in here?

Reply:
We probably can. What do you mean by sort direction though? I am only seeing the `sortDirection` param for the sort method and the `prevSortDirection` param in the setState callback. I am not sure how to turn either into its own function though